code/modules/admin/admin_verbs.dm

```diff
@@ -617,14 +617,40 @@
 	if(!istype(A))
 		return
 
-	var/turf/T = get_turf(A)
-	N.forceMove(T)
+	var/turf/T = A.jump_to_by(N)
```

Comment:
Just replace this with pick(get_area_turfs(area)) instead of making 3 shitcode procs